Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
+
+gem 'slather'
```

Comment:
I think I'm fine with the Travis setup depending on `slather` without adding a Gemfile to the repo. I'll defer to the maintainers, of course.